dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java

```diff
@@ -393,18 +392,17 @@ private String buildShellCommandFile(String jobConfigFilePath, Map<String, Prope
         }
 
         // datax python command
-        StringBuilder sbr = new StringBuilder();
```

Comment:
May I ask why you change to use `String` here?

Reply:
@EricGao888 @fuchanghai Sure, because JVM internally compiler will geneate axactly the same bytecode and `String +` and `StringBuilder.append` will have the same performance and the same memory usage, but String will be easier to read. We can get a readability benefit without any performace compromises. The replacement does not result in significant performance drawback on modern JVMs.
Please check the following screenshot, the modern JVM will pptimize `String +` to `StringBuilder.append`, so they are same.
![image](https://user-images.githubusercontent.com/20518339/196035542-fabdf90b-2b3c-4efe-882d-d05c57a76c8e.png)
If just compare `String` with `StringBuilder`, on most cases, using String is more practical and logical. It is not always better to concatenate string with plus sign (+). For example, StringBuilder's append method is more logical if you change your string in a loop because of its mutability. 
For more details, please check [JLS: 15.18.1. String Concatenation Operator +](https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.18.1)
And current IDEA is so smart, it will suggests you to change it for better code readability. It is equivalent to the optimization of the compiler, so IDEA detects that the two are equivalent. In order to make the code more concise, it will give some tips. thx

Reply:
Make sense to me. Nice work and thx for the explanation.